icons/apache/apache-line-wordmark.svg

```diff
@@ -1 +1 @@
5.7853 3.31477 75.7874C3.17918 75.79 3.01456 75.783 2.83074 75.7638C2.8251 75.7632 2.81946 75.7626 2.81379 75.762C2.36717 75.7139 1.81048 75.5949 1.28237 75.3718C1.55243 75.625 1.83269 75.8575 2.12922 76.0532C2.43087 76.2522 2.74937 76.413 3.09112 76.5188C3.1296 76.5307 3.16839 76.5419 3.20747 76.5524C3.21568 76.5546 3.22389 76.5568 3.23212 76.5589C2.84424 76.5972 2.46072 76.7231 2.07405 76.9122C1.56481 77.1612 1.05011 77.5198 0.512811 77.9318C1.38602 77.5878 2.21234 77.3263 2.9673 77.2877C2.97563 77.2873 2.98394 77.2869 2.99225 77.2865C3.00943 77.2857 3.02657 77.2851 3.04367 77.2846C3.13137 77.2818 3.21809 77.2822 3.30378 77.2859C3.34971 77.2879 3.39535 77.2909 3.44069 77.2948ZM4.29272 75.1359C4.60265 74.2757 4.92351 73.3991 5.2547 72.5104C7.35363 66.8784 9.87819 60.7518 12.673 55.2262C14.3689 51.8732 16.1537 48.7148 18.0339 46.0313C19.0107 44.6373 20.0008 43.3609 21.0588 42.2428C21.347 41.9384 21.8265 41.9226 22.1341 42.2074C22.4416 42.4923 22.4625 42.9716 22.181 43.2822C21.2591 44.2992 20.3932 45.5303 19.4773 46.9401C17.7395 49.6151 15.9731 52.8066 14.3434 56.1575C11.6577 61.68 9.27118 67.5947 7.52197 72.2908C7.56802 72.2734 7.61632 72.2603 7.66643 72.2519C8.53308 72.1072 11.0922 71.5009 13.4177 68.6119C14.6953 67.0248 15.6057 65.0164 16.2819 63.1475C16.0764 63.1916 15.876 63.2266 15.6907 63.2515C15.307 63.3032 14.9453 63.06 14.8482 62.6853C14.7512 62.3105 14.9495 61.9224 15.31 61.7813C15.9324 61.5378 16.4407 61.201 16.8578 60.8241C16.8192 60.8307 16.7799 60.8371 16.7401 60.8434C16.3519 60.9048 15.9803 60.6619 15.8807 60.2816C15.7812 59.9014 15.9862 59.5076 16.3547 59.3709C17.1672 59.0696 17.9644 58.4696 18.4633 57.8062C18.7087 57.48 19.0342 56.8744 19.3304 56.1855C19.3793 56.0718 19.4264 55.9581 19.4714 55.8453C19.0754 55.9604 18.6695 56.0506 18.3332 56.0912C17.9576 56.1365 17.6054 55.9005 17.5045 55.5359C17.4036 55.1714 17.5843 54.7878 17.9297 54.6335C17.9622 54.619 17.9958 54.6042 18.0305 54.5889C18.6089 54.3341 19.4731 53.9535 20.3691 52.6236C21.9939 50.2121 23.1829 47.5169 23.5194 45.7099C23.8378 44.0002 23.5844 42.8108 23.194 42.0426C23.0512 41.7617 22.7854 41.4211 22.462 41.1571C22.1291 40.8854 21.8278 40.7685 21.6143 40.7685C21.4815 40.7685 21.2835 40.8239 20.9635 41.052C20.6417 41.2813 20.2747 41.6276 19.8076 42.0999C18.9002 43.0174 18.3426 43.6939 17.8549 44.3078C16.1807 46.4151 14.1528 49.2142 12.1811 52.3762C9.03342 57.4242 6.06446 63.4163 4.88715 69.3122C4.64613 70.5191 4.47754 71.7115 4.40103 72.8936C4.37999 73.2187 4.15538 73.4945 3.84197 73.5813C4.04413 74.0814 4.21241 74.6138 4.29272 75.1359ZM7.12175 73.3741C7.11465 73.3935 7.10755 73.4128 7.10047 73.4322C6.68369 74.5706 6.30671 75.6272 5.98062 76.5651C7.11007 76.5456 8.13238 76.3616 8.98019 75.9381C8.69331 75.8755 8.4615 75.6511 8.39645 75.3548C8.31943 75.0039 8.49694 74.6468 8.82312 74.4963L8.82858 74.4938C9.28675 74.2824 9.74138 74.0727 10.194 73.7843C9.43257 73.9175 8.59834 73.9159 7.66677 73.7607C7.42862 73.7211 7.23152 73.574 7.12175 73.3741Z" clip-rule="evenodd"/></svg>
```

Comment:
Two path elements again (should be one) ;)

Reply:
Done

Reply:
Actually... no. That broke icon. Doh